.nvmrc

```diff
@@ -1 +1 @@
-14
+14.17
```

Comment:
That's interesting, so Jest requires a minimum specific minor version of Node? 🤔

Also, worth updating `CONTRIBUTING.md`, it has also mention of Node 14 version.

Reply:
Apparently so, or at least lists it as a dependency and `yarn` wants to enforce it.

Will update that file, thanks for the reminder.